crates/graphql/src/schema/objects/nft.rs

```diff
@@ -589,14 +589,13 @@ impl Collection {
             )
             .filter(metadata_collection_keys::collection_address.eq(self.0.mint_address.clone()))
             .filter(metadatas::burned_at.is_null())
-            .filter(metadata_collection_keys::verified.eq(true))
             .select(current_metadata_owners::owner_address)
-            .distinct()
-            .count()
-            .get_result::<i64>(&conn)
-            .context("Failed to load collection holder count")?
+            .distinct_on(current_metadata_owners::owner_address)
+            .load::<String>(&conn)
+            .context("Failed to load collection holder count")?;
+
+        i64::try_from(unique_holders.len())?
```

Comment:
https://github.com/diesel-rs/diesel/blob/master/diesel_tests/tests/group_by.rs

diesel does have group_by.

Also this seems like one where a data loader would be a good additions for this query in the case holder_count is requested from a list request.

Reply:
> /// This trait is not yet part of Diesel's public API. It may change in the
/// future without a major version bump.
///
/// This trait exists as a stop-gap for users who need to use `GROUP BY` in
/// their queries, so that they are not forced to drop entirely to raw SQL. The
/// arguments to `group_by` are not checked, nor is the select statement
/// forced to be valid.

A long, long time ago, in a PR far away, I was told not to use `group_by` because it's not a standard API feature for the version of diesel we're using. Has that policy changed? If so, we many queries that can be converted to checked queries.

Reply:
> Also this seems like one where a data loader would be a good additions for this query in the case holder_count is requested from a list request.

How about I just put all of these in async data loaders?

Reply:
I cannot get diesel to do the simplest version of this, and out of time for now. If this approach is unacceptable, we'll have to get more folks on it.